test/common/alerts_firing.go

```diff
@@ -61,10 +76,99 @@ func (e *alertsFiringError) isValid() bool {
 	return !e.deadMansSwitchFiring || len(e.alertsFiring) != 0 || len(e.alertsPending) != 0
 }
 
-// TestIntegreatlyAlertsFiring reports any firing or pending alerts
 func TestIntegreatlyAlertsFiring(t *testing.T, ctx *TestingContext) {
```

Comment:
Do you think we could add some comments to each of these functions just explaining why we have them and why they were broken out into two separate functions? I guess something similar to @david-martin's comment. I think it would just make it more clear for anyone adding to this in the future!

Reply:
Sure ill add some comments